static/app/utils/metrics/index.tsx

```diff
@@ -447,3 +449,11 @@ export function getMetaDateTimeParams(datetime?: PageFilters['datetime']) {
 
   return {statsPeriod: '14d'};
 }
+
+export function areResultsLimited(response: MetricsQueryApiResponse) {
+  const lastMetaEntries = response.meta.map(
+    meta => meta[meta.length - 1]
+  ) as MetricsQueryApiResponseLastMeta[];
+
+  return lastMetaEntries.some(meta => meta.has_more);
+}
```

Comment:
nit: could be done in one iteration
```suggestion
export function areResultsLimited(response: MetricsQueryApiResponse) {
  return response.meta.some(
    meta => (meta[meta.length - 1] as MetricsQueryApiResponseLastMeta).has_more
  );
}
```